.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: go
 go:
-  - 1.7.6
   - 1.8.3
   - 1.9
```

Comment:
Mind changing this to 1.9.2? I think that's the go version other repos use as well.

Reply:
Just curious, why remove 1.7.6?

Reply:
it has different lint rules and is no longer supported. So lint things were failing so I just got rid of it.